index.js

```diff
@@ -18,3 +20,170 @@ closeBtn.addEventListener('click', closeButton);
+    descriptionCard:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.",
+    technology1: 'html',
+    technology2: 'bootstrap',
+    technology3: 'ruby',
+    technology4: 'Ruby on rails',
+    button: 'see project',
+    btnCard1: 'See Live',
+    btnCard2: 'See Source',
+  },
+  {
+    name: 'Professional Art Printing Data',
+    nameCard: 'Multi Post Stories',
+    number: 7,
+    Description:
+      "A daily selection of privately personalized reads; no account or sign-ups required,has been the industry's standard",
+    descriptionCard:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.",
+    technology1: 'html',
+    technology2: 'bootstrap',
+    technology3: 'ruby',
+    technology4: 'Ruby on rails',
+    button: 'see project',
+    btnCard1: 'See Live',
+    btnCard2: 'See Source',
+  },
+];
+
+const cardsHtml = function () {
+  for (let index = 0; index < arrProject.length; index += 1) {
+    allWorks.innerHTML += `
+              <div class="work-${arrProject[index].number} work-common-class">
+                <div class="work-detail">
+                  <div class="main-detail">
+                    <h4>${arrProject[index].name}</h4>
+                    <p class="paragraph">
+                      ${arrProject[index].Description}
+                    </p>
+                    <div class="work-tools work-stack">
+                      <ul>
+                     <li>${arrProject[index].technology1}</li>
+                     <li>${arrProject[index].technology2}</li>
+                     <li>${arrProject[index].technology3}</li>
+                      </ul>
+                    </div>
+                  </div>
+
+                  <div class="see-project">
+                    <a href="#/" class="large-button hide-button" onclick=getModal(${index})>${arrProject[index].button}</a>
+                  </div>
+                </div>
+              </div>`;
+  }
+};
+
+window.addEventListener('load', cardsHtml);
+
+// eslint-disable-next-line
```

Comment:
- [ ] As requested by the previous code reviewer, it will be more professional to remove this comment that attempt to disable linters, and try to see if his suggested approach can fix the lining error you may be getting. Linters are meant to enforce common industry best practices in real work environments. :pray: